src/ocamlorg_frontend/pages/industrial_users.eml

```diff
@@ -186,7 +189,11 @@ the community and learn more about how they use OCaml."
       </div>
     </div>
   </div>
+  <div class="prose max-w-none  pl-12">
+    If you want to contribute to a new success story, check out the <a href="https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#story" target="_blank">Contributing Guide</a> on GitHub.
 </div>
+</div>
+ 
```

Comment:
move one section higher up, below the Case Studies (these are the success stories).